pkg/debugapi/balances.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.package debugapi
+
+package debugapi
+
+import (
+	"fmt"
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+	"sort"
+
+	"net/http"
+)
+
+type balanceResponse struct {
+	Peer    string `json:"peer"`
+	Balance int64  `json:"balance"`
+}
+
+type balancesResponse struct {
+	Balances []balanceResponse `json:"balances"`
+}
+
+type balancesResponseWErr struct {
```

Comment:
it would be cool if we can just return an error if there was one, and not have to maintain separate contracts. if there's an error return an error, if it was ok return the result